Routes/admin/index.js

```diff
@@ -0,0 +1,160 @@
+const express = require('express');
```

Comment:
I'm not really sure if we need completely separate endpoint for admin as some of the routes (mostly 'get') are duplicates of existing public routes and doesn't require access restrictions. 
I'm leaving this up to you though. Maybe we should review this later when more use cases are implemented.